ui/src/app/views/invitation-key/component.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import InvitationKeyReact, {InvitationKeyProps} from './component';
+
+import {AlertDanger} from 'app/components/alert';
+import {Button} from 'app/components/buttons';
+import {FormInput} from 'app/components/inputs';
+import {profileApi, registerApiClient} from 'app/services/swagger-fetch-clients';
+import {ProfileApi} from 'generated/fetch';
+import {ProfileApiStub} from 'testing/stubs/profile-api-stub';
+
+describe('InvitationKeyComponent', () => {
+  let props: InvitationKeyProps;
+  const onInvitationVerify = jest.fn();
+
+
+  const component = () => {
+    return mount<InvitationKeyReact>
+    (<InvitationKeyReact {...props}/>);
+  };
+
+  beforeEach(() => {
+    props = {onInvitationKeyVerify: onInvitationVerify};
```

Comment:
(1) why is the test property named differently?
(2) if they had the same name, this could be simplified to just: `props = {onInvitationKeyVerify}`